### DISCUSSION TITLE
failed to flush chunk

### DISCUSSION THREAD

**Discussion by mirisu2** [fluent/fluent-bit#5722] [Q&A]
Hi,
I have a problem with losing logs.
The logs from one of pods (kubernetes) tell me:
`failed to flush chunk '1-1657872234.995539339.flb', retry in 9 seconds: task_id=14, input=emitter_for...`
I can to find out about the reason 'failed to flush chunk' ? 
And what does it mean 'failed to flush chunk' ?

**Comment by agup006** [comment]
Hi @mirisu2 , failed flush means that Fluent Bit was unable to send the data to the specified output plugin. The most likely scenario for this is generally a broken connection between Fluent Bit and the backend that you are using. Log messages following the failure to flush the chunk are helpful in diagnosing this further if you have those available
